docs/running-tests.md

```diff
@@ -0,0 +1,18 @@
+# Run tests
```

Comment:
I think we could delete this file, as we have instruction about how to run tests in the `setup-development.md`